src/design-system/color/palettes.ts

```diff
@@ -400,8 +400,14 @@ export const backgroundColors: Record<BackgroundColor, BackgroundColorValue | Co
     },
   },
   'swap (Deprecated)': {
-    color: deprecatedColors.swapPurple,
-    mode: 'darkTinted',
+    dark: {
+      color: deprecatedColors.swapPurple,
+      mode: 'darkTinted',
+    },
+    light: {
+      color: deprecatedColors.swapPurple,
+      mode: 'darkTinted',
+    },
```

Comment:
is this going to have any repercussions elsewhere in the app?

Reply:
I just stardardized the api, kept the same values, should behave the same